src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/Application.java

```diff
@@ -8,7 +8,12 @@
 
 @SpringBootApplication
 @EnableCircuitBreaker
-@EnableFeignClients
+@EnableFeignClients(
+    basePackages = {
```

Comment:
Application wasn't able to detect IDAM client when its base package wasn't specified. No idea how else to fix it.